src/handleActions.js

```diff
@@ -3,16 +3,18 @@ import reduceReducers from 'reduce-reducers';
 import invariant from 'invariant';
 import handleAction from './handleAction';
 import ownKeys from './ownKeys';
+import { flattenReducerMap } from './namespaceActions';
 
 export default function handleActions(handlers, defaultState) {
```

Comment:
Was thinking about it earlier, but left it out, as it made readability worse, as you now have to objects after each other and on a quick glimpse you might think `{ namespace }` would be default state.

Just added tests for it, seems to work "out of the box" by just passing the (maybe `undefined`) `namespace` property through.